src/docs/sphinx/dev_guide/updating_tpls.rst

```diff
@@ -0,0 +1,192 @@
+.. ## Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _tpls-label:
+
+*********************
+Third-party Libraries
+*********************
+
+Axom dependencies are grouped into four categories: Git submodules,
+built-in TPLs in the Axom source tree, system-level TPLs, and other 
+TPL libraries. The following sections describe how to install and update 
+these dependencies for Axom.
+
+  * How does one add a new compiler or platform to the mix?
+  * How does one build a new set of TPLs for a single platform or compiler
+    for testing?
+  * What is the procedure for changing versions of one or more TPLs?
+  * How do we keep things straight when using different TPL versions for 
+    different branches?
+  * How to use the scripts for team TPL support vs. local development 
+    and experimentation?
+  * Others?
+
+
+=============
+Updating TPLs
+=============
+
+Git submodules
+--------------
+
+Currently, Axom uses three external packages that appear in the repo
+as Git submodules. These are the following, including the location of the
+package in the Axom source tree:
+
+  * `BLT <https://github.com/LLNL/blt.git>`_, which is the CMake-based build
+    system we use. Location: ``axom/src/cmake/blt``.
+  * `Axom Data <https://github.com/LLNL/axom_data.git>`_, which is where we
+    maintain data files used in testing Axom. Location: ``axom/data``.
+  * `Uberenv <https://github.com/LLNL/uberenv.git>`_, which contains Python
+    scripts we use to help automate building third-party dependencies for
+    development and deployment. Location: ``axom/scripts/uberenv``.
+
+There is no software installation process for these dependencies in the 
+traditional sense. To update one of these packages in Axom, simply go into
+its directory in Axom and check out a new version. If a version is intended
+to be changed in the Axom repo, make the version change on a branch and 
+submit a GitHub pull request as you would do for other software changes.
+More info on :ref:`toolkitbuild-label`.
+
+Built-in TPLs
+-------------
+
+Axom several lightweight header-only libraries that we use internally and
+expose for downstream customers to use if they wish.
+
+  * `CLI11 <https://github.com/CLIUtils/CLI11>`_, is a command line parser
+    for C++11 and beyond that provides a rich feature set with a simple and
+    intuitive interface.
+  * `fmt <https://github.com/fmtlib/fmt>`_, is an open-source formatting
+    library providing a fast and safe alternative to C stdio and C++ iostreams.
+  * `sol <https://github.com/ThePhD/sol2>`_, is a C++ library binding to Lua.
+  * `Sparsehash <https://github.com/sparsehash/sparsehash>`_, contains several
+    hash-map implementations.
+
+.. note:: Axom patches all built-in TPLs to be under the `axom` namespace.
```

Comment:
Replace `foo` (single quotes) with ``foo`` (double quotes) when it is an acutal software construct; e.g., namespace, directory/file name, etc. Here and other places where we see them.

Reply:
hadn't gotten to this yet (bulk move before reworking). updated.